.eslintrc

```diff
@@ -1,18 +1,9 @@
 {
   "parser": "babel-eslint",
-  "rulePaths": ["build-system/eslint-rules/"],
   "plugins": [
     "eslint-plugin-swg-internal",
     "eslint-plugin-google-camelcase"
   ],
-  "ecmaFeatures": {
-    "modules": true,
-    "arrowFunctions": true,
-    "blockBindings": true,
-    "forOf": false,
-    "destructuring": true,
-    "spread": true
```

Comment:
Presumably closure polyfills all this for IE compatibility right?

Reply:
Good question! I'll look into it